index.html

```diff
@@ -67,227 +78,257 @@ <h2 class="my-position">I’am a Software Developer</h2>
+                        </li>
+                        <li class="language-list-content">
+                            RSPec
+                        </li>
+                        <li class="language-list-content">
+                            CapyBara
+                        </li>
+                        <li class="language-list-content">
+                            Selenium
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <div class="skill">
+                <div class="language skills">
+                    <img alt="Vector" class="orange-vector" src="css/Icons/orange_vectors/circle.png">
+                    <h2 class="lang-h2">Skills</h2>
+                </div>
+                <div class="language-names">
+                    <ul class="language-list">
+                        <li class="language-list-content">
+                            Codekit
+                        </li>
+                        <li class="language-list-content">
+                            Github
+                        </li>
+                        <li class="language-list-content">
+                            Codepen
+                        </li>
+                        <li class="language-list-content">
+                            Gitlab
+                        </li>
+                        <li class="language-list-content">
+                            Terminal
+                        </li>
+                    </ul>
+                </div>
             </div>
         </div>
     </section>
     <section class="lastpage">
-        <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
-            please get in touch.</h2>
-        <form class="form" action="https://formspree.io/f/xnqwwrzy" method="POST">
-            <input class="fullname" type="text" name="fullname" placeholder="Full name" required>
-            <input class="fullname" type="email" name="fullname" placeholder="Email address" required>
-            <textarea class="text-area" placeholder="Enter text here" required></textarea>
-            <button type="submit" class="button1 button1-text button-end">Get in toucht</button>
-        </form>
+        <div class="contact-form">
+            <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
+                please get in touch.</h2>
+            <form class="form" action="https://formspree.io/f/xnqwwrzy" method="POST">
+                <input class="fullname" type="text" name="fullname" placeholder="Full name" required>
+                <input class="fullname" type="email" name="fullname" placeholder="Email address" required>
+                <textarea class="text-area" placeholder="Enter text here" required></textarea>
+                <button type="submit" class="button1 button1-text button-end">Get in toucht</button>
+            </form>
```

Comment:
- [ ] Kindly make sure you add some `paddings on inputs field text` to make your form clear and match to the Figma design. 
- [ ] As from the Figma design the `Get in touch button` is supposed to be aligned at `left`, make sure to fix that issue and correct the typing error where you wrote `Get in toucht` instead of `Get in touch`. I know you can do it :muscle: :muscle: 
### Figma design
![y](https://user-images.githubusercontent.com/83042742/171961396-dcfa847e-41f4-4ea9-9c4c-3f641540d6ba.png)

### Your design
![Y](https://user-images.githubusercontent.com/83042742/171961172-af924ce6-0a2e-4799-b458-916976248a70.png)